gomint-api/src/main/java/io/gomint/command/CommandOutput.java

```diff
@@ -11,122 +11,81 @@
 import java.util.Arrays;
 import java.util.Collection;
 import java.util.Collections;
+import java.util.function.Consumer;
 
 /**
+ * All output from a command should be collected in the provided {@link CommandOutput} instance so the client can
+ * process it in the proper manner. Sending chat messages for the command result is NOT recommended and should NEVER be
+ * done.
+ * <br><br>
+ * <b>You need to call {@linkplain #markFinished()} when your command has finished.</b>
```

Comment:
Not a big friend of markFinished() what happens when i don't call it? Why is it needed?

Reply:
To allow finishing command execution in a different thread. It is just a draft.

Reply:
I have an idea to change it so it does not need to be called by default.

Reply:
Are you satisfied with the command api changes in 4df9843?

Reply:
yes